players/rikku.py

```diff
@@ -1,20 +1,46 @@
 import logging
 
-import memory
+import memory.main
+import vars
 import xbox
 from players.base import Player
 
+game_vars = vars.vars_handle()
+
 logger = logging.getLogger(__name__)
 
 
+def omnis_items():
```

Comment:
Might make sense to put this on the class instead. I don't get strongly enough to push it though

Reply:
Yeah, the reason I didn't is that it's still used in the main battle file. It should be done later.